backend/src/routes/auth.ts

```diff
@@ -0,0 +1,70 @@
+import express from 'express'
+import { body, validationResult, CustomValidator } from 'express-validator'
+import {User, UserSchema} from '../database/models/user'
+import * as bcrypt from 'bcrypt'
+import * as jwt from 'jsonwebtoken'
+import * as fs from 'fs'
+
+const auth = express.Router();
+
+const isValidRole: CustomValidator = value => {
+    return value === "teacher" || value === "student";
```

Comment:
zrobilbym enuma z rolami i porownal typ, chyba bezpieczniej niz string

Reply:
masz rację, ma to dużo więcej sensu

Reply:
ok, dodałem enuma